include/flx/flx.hpp

```diff
@@ -7,6 +7,167 @@
 //==================================================================================================
 #pragma once
 
+//==================================================================================================
+//! @addtogroup functions
+//! @{
+//! @defgroup fluxion functions
+//! @brief first order derivation of Mathematical functions
+//!
+//! This module provides implementation for
+//! scalar and SIMD versions of mathematical functions derivative and differential.
+//!
+//! **Required header:** @code{.cpp} #include <flx/flx.hpp> @endcode
+//!
+//! @}
+//==================================================================================================
+
+//==================================================================================================
+//! @addtogroup fluxion
+//! @{
+//!    'fluxion' is the name used by Isaac newton to denote what is now more often called derivatives
```

Comment:
Isn't it the same text than the .md ? One is enough

Reply:
I don't know which one I must keep 
all the way I have never been able to see any result of either in the compiled doc